test/suite/test_tiered18.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+import inspect, os, wiredtiger, wttest
+from helper_tiered import get_auth_token, get_bucket_name, TieredConfigMixin, gen_tiered_storage_sources
+from wtscenario import make_scenarios
+
+FileSystem = wiredtiger.FileSystem  # easy access to constants
+
+class test_tiered06(wttest.WiredTigerTestCase, TieredConfigMixin):
+
+    tiered_storage_sources = [
+        ('azure_store', dict(is_tiered = True,
+            is_local_storage = False,
+            auth_token = get_auth_token('azure_store'), 
+            bucket = get_bucket_name('azure_store', 0), 
```

Comment:
Can we place the bucket names we are currently using in our personal accounts here? Or create a new bucket name uses primary for testing?

Reply:
addressed in https://github.com/wiredtiger/wiredtiger/pull/8707/commits/2293e04b33409f7e91539c11a17b4643ccb29140